app/controllers/general_reports_controller.rb

```diff
@@ -41,9 +41,13 @@ def email_to_address
   end
 
   def generate_report_data_csv
-    ExportRawReportMailer.delay.raw_report_email(email_to_address, raw_report)
-    flash[:notice] = I18n.t('controllers.reports.mail_queued', email: email_to_address)
-    redirect_to send("#{action_name}_facility_general_reports_path", current_facility)
+    if Rails.env.development?
```

Comment:
Just curious: is the motivation behind this that testing Rails mail delivery, especially with attachments, kind of sucks? I've heard good things about https://github.com/ryanb/letter_opener but haven't tried it yet.